packages/components/psammead-most-read/src/Rank/index.jsx

```diff
@@ -82,58 +82,65 @@ const isFiveOrTen = ({ listIndex, service, numberOfItems }) => {
 
 const OneColumnWrapper = styled.div`
   @media (max-width: ${GEL_GROUP_0_SCREEN_WIDTH_MAX}) {
-    min-width: ${props =>
-      listHasDoubleDigits(props.numberOfItems)
-        ? getRankMinWidth(props).group0WithOneColumn
-        : getRankMinWidth(props).group0};
+    flex: 0 0
+      ${props =>
+        listHasDoubleDigits(props.numberOfItems)
+          ? getRankMinWidth(props).group0WithOneColumn
+          : getRankMinWidth(props).group0};
   }
   @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
-    min-width: ${props =>
-      listHasDoubleDigits(props.numberOfItems)
-        ? getRankMinWidth(props).group1WithOneColumn
-        : getRankMinWidth(props).group1};
+    flex: 0 0
+      ${props =>
+        listHasDoubleDigits(props.numberOfItems)
+          ? getRankMinWidth(props).group1WithOneColumn
+          : getRankMinWidth(props).group1};
```

Comment:
Can you explain how this behaves differently to `min-width`? From the flex docs it seems to me you have configured the rank to:
- not grow
- not shrink
- have a min width as calculated in the function?

Reply:
Thank you for the review @andrewscfc. By setting flex-grow and flex-shrink to `0` the text will grow/shrink relative to the `flex-basis` of the rank . `min-width` alone on a flex child is the reason the overlap happens.